compiler/fir/analysis-tests/testData/resolve/catchParameter.kt

```diff
@@ -1,4 +1,5 @@
 // RUN_PIPELINE_TILL: FRONTEND
+// LANGUAGE: +AllowReifiedTypeInCatchClause
```

Comment:
Please add an additional test with this feature disabled to check that the error is still reported for 2.2 and lower

Reply:
There is `compiler/testData/diagnostics/tests/controlStructures/catchReifiedTypeForbidden.kt`

Reply:
Oh, indeed. I've missed it